content/en/docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -158,12 +158,9 @@ globalDefault: false
 description: "This priority class should be used for XYZ service pods only."
 ```
 
-### Non-preempting PriorityClasses (alpha) {#non-preempting-priority-class}
+## Non-preempting PriorityClasses (alpha) {#non-preempting-priority-class}
```

Comment:
Hello @dethi , Thanks for this pull request.
- Since this heading was formerly a heading level 3, does it make sense to decrease the
heading level for the code example (line 194)?
- Do you think the heading needs the `(alpha)` qualifier?
Non-preempting PriorityClasses {#non-preempting-priority-class}

Reply:
Make sense to me. Updated.

Reply:
What about `s/Non-preempting PriorityClasses/Non-preempting PriorityClass/` to match the other `PriorityClass` section?

Reply:
I suppose it makes sense to use the singular, `Non-preempting PriorityClass`